guides/api.md

```diff
@@ -58,7 +58,22 @@ curl --request GET \
   --url "$ACCESS_URL/$API_ROUTE/metrics/usage/active-users?\
```

Comment:
I wonder if we should still promote this endpoint. It does something similar :man_shrugging:

Reply:
We can remove it :)